<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>677</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>25</number>
    </property>
    <property name="topMargin">
     <number>25</number>
    </property>
    <property name="rightMargin">
     <number>25</number>
    </property>
    <property name="bottomMargin">
     <number>25</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Select input directory for uncorrected images</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonInputFolder">
        <property name="text">
         <string>Choose Folder</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditInputFolder"/>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>12</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Select input calibration file</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonCalibrationFile">
        <property name="text">
         <string>Choose Calibration File</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditCalibrationFile"/>
    </item>
    <item>
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>12</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Select output directory for corrected images</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonOutputFolder">
        <property name="text">
         <string>Choose Folder</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditOutputFolder"/>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::MinimumExpanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>15</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="labelMessage">
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonGo">
        <property name="text">
         <string>Process</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>lineEditInputFolder</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>UpdateWidgets()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>368</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>514</x>
     <y>380</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditCalibrationFile</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>UpdateWidgets()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>442</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>424</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditOutputFolder</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>UpdateWidgets()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>531</x>
     <y>236</y>
    </hint>
    <hint type="destinationlabel">
     <x>567</x>
     <y>501</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonGo</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>OnButtonProcess()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>329</x>
     <y>312</y>
    </hint>
    <hint type="destinationlabel">
     <x>327</x>
     <y>557</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonInputFolder</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>OnButtonInputFolder()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>599</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>848</x>
     <y>35</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonCalibrationFile</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>OnButtonCalibrationFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>642</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>830</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonOutputFolder</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>OnButtonOutputFolder()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>626</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>844</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>UpdateWidgets()</slot>
  <slot>OnButtonProcess()</slot>
  <slot>OnButtonOutputFolder()</slot>
  <slot>OnButtonInputFolder()</slot>
  <slot>OnButtonCalibrationFile()</slot>
 </slots>
</ui>
